### ISSUE TITLE
Encountered an error undefined Property 'rationale' doesn't exist when calling getCurrentPosition on Android

### ISSUE THREAD

**Issue by roycechua** [douglasjunior/react-native-get-location#35]
I encountered a bug when I ran getCurrentPosition on Android on the latest version 3.0.0 even though I passed a rationale object. I submitted a PR request for this issue here [https://github.com/douglasjunior/react-native-get-location/pull/34](https://github.com/douglasjunior/react-native-get-location/pull/34)

- How did you link to the library (autolinking, manual, rn link, cocoapods)?
`Autolinking`
- What version of React Native?
`0.71.1`
- What version of the library?
`3.0.0`
- iOS/Android version?
```buildToolsVersion = "33.0.0"
minSdkVersion = 21
compileSdkVersion = 33
targetSdkVersion = 33
ndkVersion = "23.1.7779620"
```
- Did the problem happen after updating React Native?
`No`
- Are you using the library for the first time?
`Yes`
- It's a bug? Provide a link to a minimal reproduction case.
`Yes, it's a bug`
1. Install the package on the React Native project
2. Request Location permission from the user (make sure that permission is granted)
3. call **getCurrentLocation** in your App (make sure to pass a rationale object)
``` javascript
GetLocation.getCurrentPosition({
      enableHighAccuracy: true,
      timeout: 60000,
      rationale: {
        title: 'Location permission',
        message: 'The app needs the permission to request your location.',
        buttonPositive: 'Ok',
      },
    })
```
4. The error should appear
<img width="385" alt="Screenshot 2023-03-06 at 10 13 42 AM" src="https://user-images.githubusercontent.com/25720149/223007940-572ffeac-59e2-4b1a-8d60-14368d8807a2.png">

**Comment by douglasjunior** [comment]
Oh sorry, I had already identified the problem and fixed it, but unfortunately, it wasn't included in the final commit.

Fixed in version 3.0.1

**Comment by roycechua** [comment]
No worries. Thanks a lot for your time Douglas. I'll check out version 3.0.1 later.
